tests/views_integration/Tribe/Events/Tickets/Views/V2/Views/List_ViewTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Tribe\Events\Tickets\Views\V2\Views;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\List_View;
+use Tribe\Test\Products\WPBrowser\Views\V2\ViewTestCase;
+
+class List_ViewTest extends ViewTestCase {
+
+	use MatchesSnapshots;
+
+	/**
+	 * Test render empty
+	 */
+	public function test_render_empty() {
+		// Sanity check
+		$this->assertEmpty( tribe_events()->found() );
+
+		$context = tribe_context()->alter(
+			[
+				'today'      => $this->mock_date_value,
+				'now'        => $this->mock_date_value,
+			]
+		);
+
+		$list_view = View::make( List_View::class, $context );
+		$html      = $list_view->get_html();
+
+		// Let's make sure the View is displaying what events we expect it to display.
+		$expected_post_ids = [];
+		$this->assertEquals(
+			$expected_post_ids,
+			$list_view->found_post_ids()
+		);
+
+		$this->assertMatchesSnapshot( $html );
```

Comment:
Be sure to commit the snapshots so they set the baseline, once you get to that point